terraform/cluster_deploy.go

```diff
@@ -200,6 +200,7 @@ func deployToLoadtestInstance(instanceNum int, instanceAddr string, loadtestDist
 		".ConnectionConfiguration.SSHKey":               remoteSSHKeyPath,
 		".ConnectionConfiguration.MattermostInstallDir": "/opt/mattermost",
 		".ConnectionConfiguration.WaitForServerStart":   false,
+		".LoadtestEnviromentConfig.NumEmoji":            0,
```

Comment:
This is required because the emoji don't work with terraform.

Reply:
Can you remind me what needs to be done to make that work? (for `mattermod`, if that ever gets back off the ground...)